README.md

```diff
@@ -1,18 +0,0 @@
-# SLEIGH
```

Comment:
If this information is out of date, can we amend it instead of removing it entirely.

Reply:
Probably the reason the readme was removed is that alessandro did a from-scratch cmake port of all of sleigh, and so the absence of the readme is just that it wasn't his focus. We should keep the readme.

Reply:
Ok, added it back in and added more detail.